runbotics/runbotics-orchestrator-ui/src/src-app/components/BotProcessRunner.tsx

```diff
@@ -126,33 +126,39 @@ const BotProcessRunner: FC<BotProcessRunnerProps> = ({
         setAnchorEl(null);
     };
 
-    const handleTerminate = async () => {
-        await dispatch(
-            schedulerActions.terminateActiveJob({ jobId: processInstance?.id })
-        )
-            .then(() => {
-                setStarted(false);
-                setLoading(false);
-                setSubmitting(false);
-                enqueueSnackbar(
-                    translate('Scheduler.ActiveProcess.Terminate.Success', {
-                        processName,
-                    }),
-                    {
-                        variant: 'success',
-                    }
-                );
-            })
-            .catch(() => {
-                enqueueSnackbar(
-                    translate('Scheduler.ActiveProcess.Terminate.Failed', {
-                        processName,
-                    }),
-                    {
-                        variant: 'error',
-                    }
-                );
-            });
+    const handleTerminate = () => {
+        if (!started) return;
+
+        const TERMINATION_DELAY = 500
+        
+        setTimeout(async () => {
```

Comment:
instead of timeout, block process termination until 1st event from is received

Reply:
done